openjdk.build/makefile

```diff
@@ -388,6 +414,270 @@ test.LambdaLoadTest
 TEST_TARGETS:= \
 $(LOAD_TESTS)
 
+# Targets to run the JCK
```

Comment:
This is a lot of extra stuff in the makefile for things that most people won't need to worry about as they won't have access to the JCK source. Perhaps move them to a separately included makefile?

Reply:
This is true.  Before we had the JCK we didn't have enough targets to be worth the extra complexity.  If we are doing this then we should really have a model of a top level makefile embedding project level makefile target include files - i.e. 
openjdk.build/makefile includes openjdk.test.jck/include/test_targets.mk and openjdk.test.load/include/test_targets.mk.  The test_targets.mk would implement their own help (make help.jck make help.load etc.).  @sxa555 , @Mesbah-Alam - are you OK with that suggestion?  We could split out the non-jck targets in a later PR.  Can you think of a better name than test_targets.mk?

Reply:
Depends if we want to go that far. With the JCK being "special" I'm kinda ok with this only been applied for the jck targets that most people won't be using and keeping the simpler case (everything in this makefile) for the other targets to keep it straightforward, so I'd prefer jck_targets.mk

Reply:
OK with the suggestion.

Reply:
I prefer having a standard approach - i.e. for any projects which defines tests to run (as opposed to just projects which create things to run as part of a load test):
1. Create <project-dir>/include/test_targets.mk containing the test.xxx targets and a test.<project>.help
2. Add an include statement for <project-dir>/include/test_targets.mk in openjdk.build/makefile
3. Add a line to the openjdk.build/makefile help target to alert users to the help.<project>.help target

Reply:
Lost some formatting in the previous comment.......

I prefer having a standard approach - i.e. for any projects which define tests to run (as opposed to just projects which create things to run as part of a load test):
1. Create &lt;project-dir&gt;/include/test_targets.mk containing the test.xxx targets and a test.&lt;project&gt;.help
2. Add an include statement for &lt;project-dir&gt;/include/test_targets.mk in openjdk.build/makefile
3. Add a line to the openjdk.build/makefile help target to alert users to the help.&lt;project&gt;.help target